examples/python/documentation/Pipfile

```diff
@@ -7,4 +7,4 @@ verify_ssl = true
 python_version = "3.7"
 
 [packages]
-cdktf = "*"
+cdktf = {path = "./../../../dist/python/cdktf-0.19.1.dev1700818458-py3-none-any.whl"}
```

Comment:
This change is probably not wanted (and I think if you use the reinstall command in the package.json not needed)

Reply:
Whoops, yeah, that must've happened automatically somehow